crates/cext/src/transpiler/passes/inverse_cancellation.rs

```diff
@@ -0,0 +1,62 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::pointers::mut_ptr_as_ref;
+
+use qiskit_circuit::circuit_data::CircuitData;
+use qiskit_circuit::converters::dag_to_circuit;
+use qiskit_circuit::dag_circuit::DAGCircuit;
+use qiskit_transpiler::passes::run_inverse_cancellation_standard_gates;
+
+/// @ingroup QkTranspilerPasses
+/// Run the InverseCancellation transpiler pass on a circuit.
+///
+/// Cancels specific gates which are inverses of each other when they occur
+/// back-to-back.
+///
+/// @param circuit A pointer to the circuit to run InverseCancellation on. This circuit
+/// pointed to will be updated with the modified circuit if the pass is able to remove any gates.
```

Comment:
This could be reworded to sound a little more 
```suggestion
/// @param circuit A pointer to the circuit to run InverseCancellation on. If the pass 
/// is able to remove any gates, the original circuit will be replaced by the resulting
/// circuit from this pass.
```

Reply:
I am pretty sure I have copy-pasted this from one of Matthew's PRs. But sure addressed in [511f2b1](https://github.com/Qiskit/qiskit/pull/14840/commits/511f2b106e631757bbdd10df40aff4f053490957), tweaking the sentence a tiny bit more.